### PR TITLE
Close #2717 Update UAlert link in Global Footer

### LINKED ISSUE #2717: Replace 'UAlert' in Global Footer with 'Annual Security Report'
Please replace the 'UAlert' link in the Global Footer with 'Annual Security Report', linking to https://clery.arizona.edu/annual-reports . 

Our Web Experience team has updated this on the Arizona.edu footer, but it also needs to be rolled out to campus via the the Global Footer.

Context from Pam Scott:
We have another change we need to make to the universal footer. In order to meet Clery Reporting guidelines, we are required to make the Clery Report information readily accessible to all applicants to the University.

Thank you!

## Conditions of satisfaction
- [ ] …
- [ ] Should ‹testable condition that should be satisfied›
- [ ] Should ‹testable condition that should be satisfied›

### PR DESCRIPTION
**Pull request by trackleft**
## Related issues
Close #2717 

## How to test

Update a site with the global footer already enabled with the defaults set.
See how the UAlert link Gets updated.

On new installs see new link.

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [x] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_global_footer/az_global_footer.install [hunk 1]
```diff
@@ -188,3 +188,36 @@ function az_global_footer_update_1020801() {
 
   return t('Updated %count X-Twitter link(s) & icon(s) in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update UAlert footer links.
+ *
+ * UAlert links in the global footer will be updated with Annual Security Report.
+ */
+function az_global_footer_update_10201001() {
```
Comment (trackleft) [inline modules/custom/az_global_footer/az_global_footer.install:197]:
```suggestion
function az_global_footer_update_1021001() {
```

File: modules/custom/az_global_footer/az_global_footer.install [hunk 2]
```diff
@@ -188,3 +188,36 @@ function az_global_footer_update_1020801() {
 
   return t('Updated %count X-Twitter link(s) & icon(s) in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update UAlert footer links.
+ *
+ * UAlert links in the global footer will be updated with Annual Security Report.
```
Comment (trackleft) [inline modules/custom/az_global_footer/az_global_footer.install:195]:
```suggestion
 * Update UAlert links in the global footer with Annual Security Report.
```

File: modules/custom/az_global_footer/az_global_footer.install [hunk 3]
```diff
@@ -188,3 +188,36 @@ function az_global_footer_update_1020801() {
 
   return t('Updated %count X-Twitter link(s) & icon(s) in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update UAlert footer links.
+ *
+ * Update UAlert links in the global footer with Annual Security Report.
```
Comment (trackleft) [inline modules/custom/az_global_footer/az_global_footer.install:195]:
```suggestion
 * Replace UAlert links in the global footer with Annual Security Report.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# How to contribute
+# How to contribute to this
```
Comment (bberndt-uaz) [inline CONTRIBUTING.md:1]: Is this an intended change for this PR?

File: modules/custom/az_global_footer/az_global_footer.install [hunk 5]
```diff
@@ -188,3 +188,36 @@ function az_global_footer_update_1020801() {
...
+    ->execute();
+
+  foreach ($menu_link_content_ids as $id) {
+    /** @var \Drupal\menu_link_content\Entity\MenuLinkContent $menu_link_content */
+    $menu_link_content = MenuLinkContent::load($id);
+    if ($menu_link_content !== NULL) {
+      // phpcs:disable
+      /* @phpstan-ignore-next-line */
+      $menu_link_content->title = 'Annual Security Report';
+      // phpcs:disable
```
Comment (bberndt-uaz) [inline modules/custom/az_global_footer/az_global_footer.install:213]: This might be extra since phpcs should be disabled on line 210?

```suggestion
```

**Review by danahertzberg — APPROVED** [review]

**Review by tadean — DISMISSED** [review]

**Review by joeparsons — DISMISSED** [review]

**Review by HenryzGreenberg — DISMISSED** [review]
Reviewed Probo: https://b384f473-f8e2-451e-9b9e-046173349ca2.probo.build/
Link renders as expected. Approved.

File: modules/custom/az_global_footer/az_global_footer.install [hunk 6]
```diff
@@ -188,3 +188,36 @@ function az_global_footer_update_1020801() {
 
   return t('Updated %count X-Twitter link(s) & icon(s) in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update UAlert footer links.
+ *
+ * Replace UAlert links in the global footer with Annual Security Report.
+ */
+function az_global_footer_update_1021001() {
```
Comment (joeparsons) [inline modules/custom/az_global_footer/az_global_footer.install:197]: If we want to backport this change to 2.9.x, we should probably change this update number.
```suggestion
function az_global_footer_update_1020901() {
```

**Review by joeparsons — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**